entrypoint.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+# Set BASH to exit upon error
+set -e
+
+# Function to check if Elasticsearch is running
+check_elasticsearch_running() {
+  if curl -s --insecure "https://localhost:9200" -o /dev/null; then
+    return 0  # Elasticsearch is running
+  else
+    return 1  # Elasticsearch is not running
+  fi
+}
+
+password_reset_elastic=false
+
+# Check if certificates have already been generated
+if [[ ! -f /home/elastic/cert_bundle.zip ]]; then
+
+  # ----------------------------
+  # CERTIFICATE GENERATION
+  # ----------------------------
+  
+  # Generate CA HTTP Certs
+  elasticsearch-${EK_VERSION}/bin/elasticsearch-certutil ca --silent --pem -out /home/elastic/ca_bundle.zip
```

Comment:
Isn't this '--out' (double dash?) Same question for all the following certutil commands.

Reply:
This has been fixed, you are correct.